.github/CONTRIBUTING.md

```diff
@@ -95,6 +95,8 @@ pnpm test
 pnpm test:typecheck
 ```
 
+Note: ensure to build the package (`pnpm build`) before running the `test:typecheck` suite.
```

Comment:
Lets change this from Note: ensure to > **Note** Ensure.

Also can you add a section in the `Install` section to make sure to use typescript version `5.0.4`

Reply:
Done in 238b89e, but a bit unsure about the change